stdlib/format.mli

```diff
@@ -418,6 +418,10 @@ val print_newline : unit -> unit
 
 (** {1 Margin} *)
 
+val pp_infinity : int
+(** [pp_infinity] is the maximal size of the margin.
+Its exact value is implementation dependent but is guaranteed to be greater than 10^9. *)
```

Comment:
10^9 ⇒ 10{^9}.
The start of the sentence below should be indented by two.

Reply:
Updated! I also added cross-references to `pp_infinity` where it is mentioned elsewhere.